generators/pythonv2/codegen/src/ast/__test__/Class.test.ts

```diff
@@ -1,11 +1,19 @@
 import { python } from "../..";
+import { Writer } from "../core/Writer";
 
 describe("class", () => {
+    let writer: Writer;
+
+    beforeEach(() => {
+        writer = new Writer();
+    });
```

Comment:
You'll find that I've updated all tests to call the writer's `toStringFormatted` method rather than adding a `toStringFormatted` method to the `AstNode`.

I describe why below.